packages/react-ui/src/index.js

```diff
@@ -12,11 +12,16 @@ import LegendProportion from './widgets/legend/LegendProportion';
 import LegendRamp from './widgets/legend/LegendRamp';
```

Comment:
Please add a new entry in the changelog with a descriptive title

Reply:
I've updated the changelog, thx @vmilan